geopandas/geodataframe.py

```diff
@@ -1696,9 +1696,11 @@ def dissolve(
 
         Parameters
         ----------
-        by : string, default None
-            Column whose values define groups to be dissolved. If None,
-            whole GeoDataFrame is considered a single group.
+        by : str or list-like, default None
+        Column(s) whose values define the groups to be dissolved. If None,
+        the entire GeoDataFrame is considered as a single group. If a list-like
+        object is provided, all attributes in the list need to be equal for polygons
+        to be combined.
```

Comment:
@martinfleis Sorry for the confusion as I am new to this I didn't understand it correctly. Can you review the below statements so that I can update my pull request.
```         
        by : str or list-like, default None
            Column(s) whose values define the groups to be dissolved. If None,
            the entire GeoDataFrame is considered as a single group. If a list-like
            object is provided, the values in the list are treated as categorical labels,
            and polygons will be combined based on the equality of these categorical labels.
            The list should have the same length as the GeoDataFrame, and its values are
            considered as categorical labels.
```

Reply:
You can drop the last sentence, the rest is okay I suppose.

Reply:
@martinfleis I have made the edit pl chk.